js/jquery.mapael.js

```diff
@@ -1880,87 +1871,129 @@
-                $(textElem.node).on("mouseout." + pluginName, outBehaviour);
+            /* Handle textElem Hover attributes */
+            if (elem.textElem !== undefined) {
+                if (elem.textElem.attrsHover.animDuration > 0) elem.textElem.animate(elem.textElem.attrsHover, elem.textElem.attrsHover.animDuration);
+                else elem.textElem.attr(elem.textElem.attrsHover);
             }
-        },
 
-        /*
-         * Remove the hover behavior for plots and areas
-         * @param mapElem the map element
-         * @param textElem the optional text element (within the map element)
-         */
-        unsetHover: function (mapElem, textElem) {
-            $(mapElem.node).off("." + pluginName);
-            if (textElem) $(textElem.node).off("." + pluginName);
+            /* Handle tooltip init */
+            if (elem.tooltip !== undefined) {
+                var content = '';
+                // Reset classes
+                self.$tooltip.removeClass().addClass(self.options.map.tooltip.cssClass);
+                // Get content
+                if (elem.tooltip.content !== undefined) {
+                    // if tooltip.content is function, call it. Otherwise, assign it directly.
+                    if (typeof elem.tooltip.content === "function") content = elem.tooltip.content(elem.mapElem);
+                    else content = elem.tooltip.content;
+                }
+                if (elem.tooltip.cssClass !== undefined) {
+                    self.$tooltip.addClass(elem.tooltip.cssClass);
+                }
+                self.$tooltip.html(content).css("display", "block");
+            }
+
+            // workaround for older version of Raphael
+            if (elem.mapElem !== undefined || elem.textElem !== undefined) {
+                if (self.paper.safari) self.paper.safari();
+            }
         },
 
         /*
-         * Set he behaviour for "mouseover" event
-         * @param mapElem mapElem the map element
-         * @param textElem the optional text element (within the map element)
+         * Set the behaviour when mouse moves in element ("mousemove" event)
+         * @param elem the map element
          */
-        elemHover: function (mapElem, textElem) {
+        elemHover: function (elem, event) {
             var self = this;
-            // Set mapElem
-            if (mapElem.attrsHover.animDuration > 0) mapElem.animate(mapElem.attrsHover, mapElem.attrsHover.animDuration);
-            else mapElem.attr(mapElem.attrsHover);
-            // Set textElem
-            if (textElem) {
-                if (textElem.attrsHover.animDuration > 0) textElem.animate(textElem.attrsHover, textElem.attrsHover.animDuration);
-                else textElem.attr(textElem.attrsHover);
+            if (elem === undefined) return;
+
+            /* Handle tooltip position update */
+            if (elem.tooltip !== undefined) {
+                console.log(elem);
```

Comment:
console.log() should be removed

Reply:
I don't see it actually... Did you check the latest rev?